### ISSUE TITLE
 W0311: Bad indentation. Found 4 spaces, expected 2 (bad-indentation)

### ISSUE THREAD

**Issue by polyglotm** [PyCQA/pylint#2749]
## Question

what does that mean??
it should be oriented 4.

```
....def __init__(self):
........super().__init__()
```
this is real code. but lint return these things
```
W0311: Bad indentation. Found 4 spaces, expected 2 (bad-indentation)
W0311: Bad indentation. Found 8 spaces, expected 4 (bad-indentation)
```

for now, I just disable W0311. but I really do not understand. why is that?

**Comment by PCManticore** [comment]
Hi, thanks for the report. Do you use by any chance tabs instead of spaces?

**Comment by polyglotm** [comment]
@PCManticore no, I use only space

![screenshot 2019-02-19_21-02-35-797](https://user-images.githubusercontent.com/13045112/53013819-f78eb700-3489-11e9-8abb-787b461841f6.png)
![screenshot 2019-02-19_21-03-04-700](https://user-images.githubusercontent.com/13045112/53013825-fb223e00-3489-11e9-95cd-8ea0b999cde3.png)

**Comment by elvijs** [comment]
Not sure if this helps, but I saw the same symptoms when I had accidentally misconfigured the pylintrc file (setting 2 spaces for indentation instead of 4).

**Comment by AWhetter** [comment]
Closing because it is close to year since this was reported but we still haven't been able to reproduce it. Please feel free to open a new issue if you come across this problem.

**Comment by richardmillson** [comment]
@polyglotm did you happen to download the `pylintrc` recommended by the [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)? It [sets `indent-string` to two instead of four spaces](https://github.com/google/styleguide/blob/b34b3b30b47f03068cab59fd91f361eac0fc026e/pylintrc#L283) with the comment

> The internal Google style guide mandates 2 spaces.  Google's externaly-published style guide says 4, consistent with PEP 8. Here, we use 2 spaces, for conformity with many open-sourced Google projects (like TensorFlow).

You can either set `indent-string='    '` in your `pylintrc` to get the behaviour you want or invoke `pylint` with
```shell
pylint --indent-string='    ' your_file.py
```
